book/src/volume-cloning.md

```diff
@@ -0,0 +1,56 @@
+# Volume Cloning
+
+## Status and Releases
+
+** Status:** Alpha
+
+Compatible with CSI Version | Supported CSI external Provisioner Version | Min k8s Version | Max k8s version
+--|--|--|--|--|--
+v1.0  | v1.3 | v1.15 | -
+
+## Overview
+
+A Clone is defined as a duplicate of an existing Kubernetes Volume.  For more information on cloning in Kubernetes see the concepts doc for [Volume Cloning](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-cloning).
+
+A storage provider that allows volume cloning as a create feature, may choose to implement volume cloning via a
+control-plane CSI RPC call.
+
+## Implementing Volume cloning functionality
+
+To implement volume cloning the CSI driver MUST:
+
+1. Implement checks for `csi.CreateVolumeRequest.VolumeContentSource` in the plugins `CreateVolume` function implementation.
```

Comment:
typo: plugin's

Reply:
pluralization of a regular noun, but I make so many typos I'm the last person to argue